### PR TITLE
Add lava-obscures to Paper configuration documentation

### PR DESCRIPTION
**Pull request by TASelwyn**
Adds lava-obscures option to the wiki documentation.

### CODE REVIEW

**Comment by Chew** [comment]
Update the "last updated" at the top as well

**Comment by TASelwyn** [comment]
> Update the "last updated" at the top as well

Ha, I didn't even know that existed 😆

**Comment by TASelwyn** [comment]
Commited to the pull request branch to fix the date @Chew

File: source/server/configuration.rst [hunk 1]
```diff
@@ -841,6 +841,10 @@ anti-xray
     - **default**: 2
     - **description**: Controls the distance in blocks from air or water that
       hidden-blocks are hidden by the anti-xray engine.
+      
+* lava-obscures
+    - **default**: false
+    - **description**: Whether or not to obfuscate blocks touching lava. (Note: Does not work well with non-stone-like ore textures)
```
Comment (Proximyst) [inline source/server/configuration.rst:847]: What does "not work well" mean here? Make it clearer or drop that part.

**Review by Proximyst — CHANGES_REQUESTED** [review]

**Review by TASelwyn — COMMENTED** [review]
fixed in a commit

**Comment by TASelwyn** [comment]
@Proximyst Do you mind reviewing the change?

**Review by Chew — APPROVED** [review]
Looks good

**Review by Proximyst — APPROVED** [review]

**Pull request merged**